src/Tests/Microsoft.NET.Publish.Tests/GivenThatWeWantToPublishAnAotApp.cs

```diff
@@ -18,6 +18,31 @@ public GivenThatWeWantToPublishAnAotApp(ITestOutputHelper log) : base(log)
         {
         }
 
+        [Fact]
+        public void Can_Publish_SingleFile_And_NativeAot()
+        {
+            var targetFramework = ToolsetInfo.CurrentTargetFramework;
+            var project = CreateHelloWorldTestProject(targetFramework, "SingleFileAndNativeAot", true);
+            var testAsset = _testAssetsManager.CreateTestProject(project);
+            var buildProperties = project.GetPropertyValues(testAsset.TestRoot, targetFramework);
+            project.RecordProperties("NETCoreSdkPortableRuntimeIdentifier");
+            project.AdditionalProperties["PublishSingleFile"] = "true";
+            project.AdditionalProperties["PublishAot"] = "true";
+
+            var rid = buildProperties["NETCoreSdkPortableRuntimeIdentifier"];
+            var publishCommand = new PublishCommand(testAsset);
+            publishCommand
+                .Execute($"/p:RuntimeIdentifier={rid}", "/p:SelfContained=true")
```

Comment:
Is `/p:RuntimeIdentifier={rid}` and `/p:SelfContained=true` still needed?

Reply:
That's a good question. I think the warning has been removed, so this might be fine. I'll try it and see.